### ISSUE TITLE
== Error ==  system call for: "['anaconda2/bin/spades-core', 

### ISSUE THREAD

**Issue by XMTian** [ablab/spades#551]
```
  0:00:13.464    62M / 361M  INFO    General                 (construction.cpp          : 150)   Max read length 150
  0:00:13.464    62M / 361M  INFO    General                 (construction.cpp          : 156)   Average read length 136.5
  0:00:13.464    62M / 361M  INFO    General                 (stage.cpp                 : 113)   PROCEDURE == k+1-mer counting
  0:00:13.980    62M / 361M  INFO    General                 (kmer_index_builder.hpp    : 117)   Splitting kmer instances into 256 files using 16 threads. This might take a while.
  0:00:14.068    62M / 361M  INFO    General                 (file_limit.hpp            :  32)   Open file limit set to 1024
  0:00:14.068    62M / 361M  INFO    General                 (kmer_splitters.hpp        :  89)   Memory available for splitting buffers: 2.60413 Gb
  0:00:14.069    62M / 361M  INFO    General                 (kmer_splitters.hpp        :  97)   Using cell size of 262144


== Error ==  system call for: "['anaconda2/bin/spades-core', '03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test1/K21/configs/config.info', '03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test1/K21/configs/isolate_mode.info']" finished abnormally, OS return value: -9

```

I have this problem, can you help me solve it?

**Comment by asl** [comment]
Sure. Will you please at least attach the files mentioned in the error message?

**Comment by XMTian** [comment]
| |
xmtian94
|
|
xmtian94@163.com
|
签名由网易邮箱大师定制
On 7/24/2020 18:03，Anton Korobeynikov<notifications@github.com> wrote：

Sure. Will you please at least attach the files mentioned in the error message?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Command line: anaconda2/bin/spades.py	-1	02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_1.fq.gz	-2	02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_2.fq.gz	-o	03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test	--isolate	

System information:
  SPAdes version: 3.14.0
  Python version: 2.7.15
  OS: Linux-2.6.32-431.el6.x86_64-x86_64-with-redhat-6.5-Santiago

Output dir: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test
Mode: ONLY assembling (without read error correction)
Debug mode is turned OFF

Dataset parameters:
  Isolate mode
  Reads:
    Library number: 1, library type: paired-end
      orientation: fr
      left reads: ['02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_1.fq.gz']
      right reads: ['02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_2.fq.gz']
      interlaced reads: not specified
      single reads: not specified
      merged reads: not specified
Assembly parameters:
  k: automatic selection based on read length
  Repeat resolution is enabled
  Mismatch careful mode is turned OFF
  MismatchCorrector will be SKIPPED
  Coverage cutoff is turned OFF
Other parameters:
  Dir for temp files: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/tmp
  Threads: 16
  Memory limit (in Gb): 125

Command line: anaconda2/bin/spades.py	-1	02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_1.fq.gz	-2	02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_2.fq.gz	-o	03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test	--isolate	

System information:
  SPAdes version: 3.14.0
  Python version: 2.7.15
  OS: Linux-2.6.32-431.el6.x86_64-x86_64-with-redhat-6.5-Santiago

Output dir: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test
Mode: ONLY assembling (without read error correction)
Debug mode is turned OFF

Dataset parameters:
  Isolate mode
  Reads:
    Library number: 1, library type: paired-end
      orientation: fr
      left reads: ['02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_1.fq.gz']
      right reads: ['02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_2.fq.gz']
      interlaced reads: not specified
      single reads: not specified
      merged reads: not specified
Assembly parameters:
  k: automatic selection based on read length
  Repeat resolution is enabled
  Mismatch careful mode is turned OFF
  MismatchCorrector will be SKIPPED
  Coverage cutoff is turned OFF
Other parameters:
  Dir for temp files: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/tmp
  Threads: 16
  Memory limit (in Gb): 125


======= SPAdes pipeline started. Log can be found here: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/spades.log

02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_2.fq.gz: max reads length: 150
02.RmAdapter/Pool_503a_maxMM3maxN2_Bacteria_strain_no_250_D704_D506.RmAdp_1.fq.gz: max reads length: 150

Reads length: 150

Default k-mer sizes were set to [21, 33, 55, 77] because estimated read length (150) is equal to or greater than 150

===== Assembling started. 


===== K21 started. 


== Running: anaconda2/bin/spades-core 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/config.info 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/isolate_mode.info

  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  73)   Loaded config from 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/config.info
  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  73)   Loaded config from 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/isolate_mode.info
  0:00:00.000     6M / 12M   INFO    General                 (memory_limit.cpp          :  49)   Memory limit set to 125 Gb
  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  83)   Starting SPAdes, built from N/A, git revision N/A
  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  84)   Maximum k-mer length: 128
  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  85)   Assembling dataset (03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/dataset.info) with K=21
  0:00:00.000     6M / 12M   INFO    General                 (main.cpp                  :  86)   Maximum # of threads to use (adjusted due to OMP capabilities): 16
  0:00:00.000     6M / 12M   INFO    General                 (launch.hpp                :  79)   SPAdes started
  0:00:00.001     6M / 12M   INFO    General                 (launch.hpp                :  86)   Starting from stage: read_conversion
  0:00:00.001     6M / 12M   INFO    General                 (launch.hpp                :  92)   Two-step RR enabled: 0
  0:00:00.001     6M / 12M   INFO   StageManager             (stage.cpp                 : 166)   STAGE == Binary Read Conversion
  0:00:00.341     6M / 14M   INFO    General                 (read_converter.cpp        :  72)   Converting reads to binary format for library #0 (takes a while)
  0:00:00.341     6M / 14M   INFO    General                 (read_converter.cpp        :  73)   Converting paired reads
  0:00:03.772   201M / 201M  INFO    General                 (binary_converter.cpp      :  96)   16384 reads processed
  0:00:03.916   221M / 222M  INFO    General                 (binary_converter.cpp      :  96)   32768 reads processed
  0:00:04.171   256M / 272M  INFO    General                 (binary_converter.cpp      :  96)   65536 reads processed
  0:00:04.739   300M / 331M  INFO    General                 (binary_converter.cpp      :  96)   131072 reads processed
  0:00:05.826   346M / 380M  INFO    General                 (binary_converter.cpp      :  96)   262144 reads processed
  0:00:06.731   345M / 383M  INFO    General                 (binary_converter.cpp      : 111)   334274 reads written
  0:00:06.731   321M / 383M  INFO    General                 (read_converter.cpp        :  81)   Converting single reads
  0:00:08.751   331M / 383M  INFO    General                 (binary_converter.cpp      : 111)   0 reads written
  0:00:08.752   331M / 383M  INFO    General                 (read_converter.cpp        :  87)   Converting merged reads
  0:00:09.730   331M / 383M  INFO    General                 (binary_converter.cpp      : 111)   0 reads written
  0:00:10.466    64M / 383M  INFO   StageManager             (stage.cpp                 : 166)   STAGE == de Bruijn graph construction
  0:00:10.686    64M / 383M  INFO    General                 (construction.cpp          : 150)   Max read length 150
  0:00:10.686    64M / 383M  INFO    General                 (construction.cpp          : 156)   Average read length 136.5
  0:00:10.686    64M / 383M  INFO    General                 (stage.cpp                 : 113)   PROCEDURE == k+1-mer counting
  0:00:11.390    64M / 383M  INFO    General                 (kmer_index_builder.hpp    : 117)   Splitting kmer instances into 256 files using 16 threads. This might take a while.
  0:00:11.637    64M / 383M  INFO    General                 (file_limit.hpp            :  32)   Open file limit set to 1024
  0:00:11.637    64M / 383M  INFO    General                 (kmer_splitters.hpp        :  89)   Memory available for splitting buffers: 2.60413 Gb
  0:00:11.637    64M / 383M  INFO    General                 (kmer_splitters.hpp        :  97)   Using cell size of 262144


== Error ==  system call for: "['anaconda2/bin/spades-core', '03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/config.info', '03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/K21/configs/isolate_mode.info']" finished abnormally, OS return value: -9

In case you have troubles running SPAdes, you can write to spades.support@cab.spbu.ru
or report an issue on our GitHub repository github.com/ablab/spades
Please provide us with params.txt and spades.log files from the output directory.

SPAdes log can be found here: 03.Assembly/SPAdes/Pool_503a_maxMM3maxN2_Bacteria_strain_no_136_D707_D503/test/spades.log

Thank you for using SPAdes!

**Comment by asl** [comment]
You're likely running out of RAM. Please reopen if the problem reproduces with SPAdes 3.15.2
